src/ComicK/ComicKParser.ts

```diff
@@ -118,86 +118,96 @@ export const parseChapters = (
     strictNameMatching: boolean,
     uploaders: string[]
 ): void => {
+    var filteredChapters = data.chapters
```

Comment:
don't use var, that is in the global name space

Reply:
not the most familiar with js, updated to let. is that correct?